bin/geo2svg

```diff
@@ -63,7 +66,9 @@ function transform(d) {
       + (v = render(d), v ? " d=\"" + v + "\"" : "")
       + ">"
       + ((v = p && p["title"]) != null ? "<title>" + text(v + "") + "</title>" : "")
-      + "</path>" + os.EOL);
+      + ("</path>" + os.EOL)
+      + ((v = p && p["label"]) != null ? ("  <text transform=\"" + translation(d) + "\" stroke-width=\"3\"" + ((v = p["stroke"]) != null ? " stroke=\"" + attr(v + "") + "\"" : "") + ((v = p["fontSize"]) != null ? " font-size=\"" + attr(v + "") + "\"" : "") + ">" + text(v = p["label"]) + "</text>" + os.EOL) : "")
+      + ((v = p && p["label"]) != null ? ("  <text transform=\"" + translation(d) + "\"" + ((v = p["fill"]) != null ? " fill=\"" + attr(v + "") + "\"" : "") + ((v = p["fontSize"]) != null ? " font-size=\"" + attr(v + "") + "\"" : "") + ">" + text(v = p["label"]) + "</text>" + os.EOL) : ""));
```

Comment:
I’d like to keep geo2svg un-opinionated (as much as possible) about how GeoJSON is translated into SVG, so my preference would be to have duplicate features with different style properties to achieve stroked halos. I’d also like point features to be able to specify:

- [x] fill
- [ ] stroke
- [ ] stroke-width
- [ ] opacity
- [x] font-size
- [ ] font-family (on a per-feature basis, overriding the default)
- [ ] text-anchor

I also wonder whether we need some mechanism to offset text labels more easily from their points. That can be done by having duplicate inputs with different coordinates, I suppose.

And likewise what if you don’t want the point to be displayed? You set the radius to zero, possibly.